### ISSUE TITLE
[FEATURE REQUEST]Game icon next to title in the Steam store 

### ISSUE THREAD

**Issue by lucasbooz** [IsThereAnyDeal/AugmentedSteam#1539]
### Current Behavior (if any)
I would like you to enhance the Steam store by adding the respective game icon next to its title, as in the image below.
### Desired Behavior
![image](https://user-images.githubusercontent.com/67797271/205519934-e61c1278-feb0-4308-9a36-420f7d96a9e8.png)
### Motivation / Use Case for Changing the Behavior
It would just be a purely cosmetic change to the Steam store.

**Comment by candela97** [comment]
Another SteamDB thing eh 😏

This is simple enough to do, but will require an option - something we have been working on decreasing. I personally feel this doesn't meet the bar in terms of usefulness. The game header image works with overrides and stuff and is a lot more descriptive of the game.

In any case, this can be achieved with a userstyle manager like Stylus, just add the following styles for `store.steampowered.com` and `steamcommunity.com`:
```css
.apphub_AppIcon {
    display: block !important;
    float: left;
    margin-right: 6px;
}
````

You can also use the cosmetic filtering ability of adblockers to write a rule for this.

**Comment by lucasbooz** [comment]
That's right, I've been doing comparisons between SteamDB extensions and Augmented Steam, and both are equally useful I won't deny it, but I particularly feel a certain discomfort when using two extensions that have the same functionality, and that's why I suggested that certain features of SteamDB were also here. I also didn't do it with any intention of discrediting SteamDB, just in my view Augmented Steam serves as an "all in one" extension.

**Comment by candela97** [comment]
Yep, that's totally fine, though I think there're people who have both extensions enabled and expect both to work, and adding this CSS will definitely cause conflicts if we do it slightly differently (visually, but still), so that's one thing to consider.
